packages/calypso-e2e/src/lib/flows/start-import-flow.ts

```diff
@@ -209,14 +226,33 @@ export class StartImportFlow {
 	 * @param {string} siteSlug The site slug URL.
 	 */
 	async startSetup( siteSlug: string ): Promise< void > {
-		const route =
-			this.framework === 'signup' ? '/start/setup-site/intent' : '/setup/site-setup/intent';
+		const route = '/setup/site-setup/intent';
 
 		await this.page.goto( DataHelper.getCalypsoURL( route, { siteSlug } ) );
 		await this.validateSetupPage();
 		await this.page.click( selectors.startImportButton );
 	}
 
+	/**
+	 * Import focused flow, go to first import step
+	 */
+	async startImportFocused( step: string, siteSlug: string, from: string ): Promise< void > {
+		const route = step ? `/setup/import-focused/${ step }` : '/setup/import-focused';
```

Comment:
nit: If we make an empty string the default value for step, the following should be OK:

```suggestion
		const route = `/setup/import-focused/${ step }`
```